src/Microsoft.AspNet.OData.Shared/Formatter/Serialization/ODataResourceSerializer.cs

```diff
@@ -275,7 +278,49 @@ private ResourceContext GetResourceContext(object graph, ODataSerializerContext
             }
         }
 
-        private async Task WriteDeltaComplexPropertiesAsync(SelectExpandNode selectExpandNode,
+        internal void WriteDeltaNavigationProperties(SelectExpandNode selectExpandNode, ResourceContext resourceContext, ODataWriter writer)
```

Comment:
Document internal method. In addition, the convention is to have internal methods above private methods